app/models/usertype.rb

```diff
@@ -0,0 +1,2 @@
+class Usertype < ActiveRecord::Base
+end
```

Comment:
can this model be deleted? it appears it's only a string. you could store the hard-coded values as a constant in user.rb